commands/utils/poller/build.js

```diff
@@ -0,0 +1,37 @@
+const constants = require("../constants");
+const request = require("request");
+//https://api.cypress-v3.dev.lambdatest.io/api/v1/test/stop/?sessionId=4a7434b9-1905-4aaf-a178-9167acb00c5d
+function stop_cypress_session(lt_config, session_id, env) {
+  return new Promise(function (resolve, reject) {
+    request(
+      constants[env].BUILD_STOP_URL + session_id,
+      {
+        auth: {
+          username: lt_config["lambdatest_auth"]["username"],
+          password: lt_config["lambdatest_auth"]["access_key"],
+        },
+        method: "PUT",
+      },
+      (err, res, body) => {
+        if (err) {
+          console.log("Error occured while stopping session", err);
+          reject(err);
+        }
+        if (res.statusCode == "401") {
```

Comment:
can we use http status code constants provided by node(if node provides) instead of hardcoding ?
what do u say ? @japneetlambdatest

Reply:
They are not provided as default just like we have in go, we will have to used one more package to add that which I don't think makes sense.
In upcoming releases we can add global constants and use them across code